lib/portable-text/portable-text-types.js

```diff
@@ -1,7 +1,9 @@
 import Image from 'next/image';
 import { Disclosure } from '@headlessui/react';
 import CustomButton from '../../components/custom-button';
-import { ChevronUpIcon } from '@heroicons/react/outline';
+import { useNextSanityImage } from 'next-sanity-image';
+import { config } from '../config';
+import { IoMdArrowDropdownCircle } from 'react-icons/io';
```

Comment:
again super small thing but lets keep convention of imports from external packages over local ones

Reply:
Oops! Will fix that up before merging